questions/qa-address-formats.en.md

```diff
@@ -0,0 +1,107 @@
+# Address formats around the world
+
+## Question
+
+### How do address formats differ around the world, and what are the implications of those differences on the design of forms, databases, ontologies, etc. for the Web?
+
+Address formats vary widely across the globe, with differences in structure, content, and the level of granularity. For authors and developers designing forms, databases, or systems that handle addresses, understanding these variations is crucial to avoid frustrating users from other countries. This article will introduce some of the key differences in address formats around the world and provide guidance on how to design systems that can handle them effectively.
+
+This is not an exhaustive guide but aims to sensitize you to the complexities of international address formats and the challenges they pose for web design. As with [personal names](https://www.w3.org/International/questions/qa-personal-names.en.html), there is rarely a "perfect" solution, but awareness of these differences is the first step toward building more inclusive systems.
```

Comment:
A difference that i think is worth mentioning is the order of street name and house number.  In Germany & Switzerland the house number follows the street name, which has implications for form design.

Also, some people may not have house numbers, or may need to add a house name, or a flat number.  This leads to the more general issue, too, that the number of lines needed for an address can vary.

There are also things like where the address should be printed on an envelope - in Switzerland i used to get in trouble for putting the 'to' address in the middle.

Reply:
Do you have any examples for the points you mentioned? I can put them in the article when I write.